source/localizable/getting-started/index.md

```diff
@@ -55,7 +63,7 @@ need for a browser to be open. Consult the [PhantomJS download instructions](htt
 Install Ember using npm:
 
 ```bash
-npm install -g ember-cli@2.4
```

Comment:
Please remove this from the PR. See https://github.com/emberjs/guides/pull/1379.